validation/superchain-version_test.go

```diff
@@ -0,0 +1,135 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+// TestContractVersions will check that
+//   - for each chain in OPChain
+//   - for each declared contract "Foo" : version entry in the corresponding superchain's semver.yaml
+//   - the chain has a contract FooProxy deployed at the same version
+//
+// Actual semvers are
+// read from the L1 chain RPC provider for the chain in question.
+func TestContractVersions(t *testing.T) {
+
+	isExcluded := map[uint64]bool{
+		10:           true,
+		291:          true,
+		420:          true,
+		424:          true,
+		888:          true,
+		957:          true,
+		997:          true,
+		8453:         true,
+		34443:        true,
+		58008:        true,
+		84531:        true,
+		84532:        true,
+		7777777:      true,
+		11155420:     true,
+		11763071:     true,
+		999999999:    true,
+		129831238013: true,
+	}
+
+	isSemverAcceptable := func(desired, actual string) bool {
+		return desired == actual
+	}
+
+	checkOPChainSatisfiesSemver := func(chain *ChainConfig) {
+		rpcEndpoint := Superchains[chain.Superchain].Config.L1.PublicRPC
+
+		if rpcEndpoint == "" {
+			t.Errorf("%s has MISSING RPC endpoint", chain.Superchain)
```

Comment:
[nit] We use [testify](https://github.com/stretchr/testify) for all of our assertions - can you please convert these to use the `require` package from Testify?

Reply:
[549d7d6](https://github.com/ethereum-optimism/superchain-registry/pull/62/commits/549d7d64b8065c1a049cb7f4d43ba8c15b793117)